Src/CSharpier.Tests/CommandLineFormatterTests.cs

```diff
@@ -448,66 +465,71 @@ public void Empty_Config_Files_Should_Log_Warning(string configFileName)
             .Be($"Warning The configuration file at {configPath} was empty.");
     }
 
-    private FormatResult Format(
-        bool skipWrite = false,
-        bool check = false,
-        bool writeStdout = false,
-        string standardInFileContents = null,
-        params string[] directoryOrFilePaths
-    )
+    private class TestContext
     {
-        if (directoryOrFilePaths.Length == 0)
+        private MockFileSystem fileSystem = new();
+
+        public string WhenAFileExists(string path, string contents)
         {
-            directoryOrFilePaths = new[] { GetRootPath() };
+            path = this.fileSystem.Path.Combine(GetRootPath(), path).Replace('\\', '/');
+            this.fileSystem.AddFile(path, new MockFileData(contents));
+            return path;
         }
-        else
+
+        public string GetRootPath()
```

Comment:
Maybe `CreateAndGetRootPath()`? It feels really weird for a method called `GetBlah()` to mutate state.

Reply:
I switched this around so it no longer mutates state.